src/planner/Planner.cpp

```diff
@@ -6,10 +6,15 @@ namespace aikido {
 namespace planner {
 
 //==============================================================================
-Planner::Planner(statespace::ConstStateSpacePtr stateSpace)
+Planner::Planner(statespace::ConstStateSpacePtr stateSpace, common::RNG* rng)
   : mStateSpace(std::move(stateSpace))
 {
-  // Do nothing
+  if (!rng)
+    mRng = std::move(
+        std::unique_ptr<common::RNG>(
+            new common::RNGWrapper<std::mt19937>(std::random_device{}())));
+  else
+    mRng = std::move(cloneRNGFrom(*rng)[0]);
```

Comment:
Nit: For readability, please use curly brackets for multiple lines of control statements like:
```cpp
 if (!rng)
 {
   mRng = std::move(
       std::unique_ptr<common::RNG>(
           new common::RNGWrapper<std::mt19937>(std::random_device{}())));
 }
 else
 {
   mRng = std::move(cloneRNGFrom(*rng)[0]);
 }
```